packages/workflow/src/records/fhir.ts

```diff
@@ -218,9 +219,21 @@ export function createDocumentReferenceEntryForCertificate(
   temporaryRelatedPersonId: UUID,
   eventType: EVENT_TYPE,
   hasShowedVerifiedDocument: boolean,
+  collectorDetails: CertifyInput['collector'],
+  currentTask: SavedTask,
```

Comment:
let's directly pass in `practitionerReference` here as an optional value and move the logic for that one level up